### ISSUE TITLE
Error: Cannot read property "startsWidth" or undefined 

### ISSUE THREAD

**Issue by pjvargo43** [sologenic/sologenic-expo-wallet#141]
This error pop up anytime I try to move, trade or sell.

Error: Cannot read property "startsWidth" or undefined

**Comment by kamokat** [comment]
Dose it to me too... And I can not connect wallet to sologenics.io comes up ERROR

**Comment by JoJoMud66** [comment]
I'm getting the same message also!! ?????

**Comment by BastienPenalba** [comment]
Same here, honestly this is a joke. Tried activating SOLO since yesterday: sometimes my funds were gone, then back up, but whatever I do i get so many errors. How are we supposed to trust this product?

**Comment by pjvargo43** [comment]
• With my passed experiences  of "Errors" on numerous different platforms, if you made a transaction and it shows up on the blockchain, your coins are safe. 

• I've used Solo DEX wallet, even before it changed over to a DEX wallet, for over a year and I never had any issues until now. I believe with everyone jumping on the band wagon, its flooding the DEX, creating these new errors Sologenic hasn't yet faced. All we can do is give them time to address the issues. 

• As an alternative, I'm using **_HotBit Exchange_** to buy my SOLO for the time being. While exchanging some of my assets to USDT, I'm able to purchase more SOLO/USDT tokens. When Sologenic's is done addressing the wallet error, then I'll move my SOLO tokens over. ***Keep in mind*** at HotBit Exchange, there's a fee for withdrawing SOLO and that is 4 SOLO tokens with every withdraw. Also, before you withdraw your SOLO tokens over to SoloDEX, go into settings and add an ID to your wallet address or HotBit will have trouble identifying where your tokens need to go. 

☆ Hope this information helped while we wait for the errors to clear. If you do not use HotBit Exchange, I assure you it's very user friendly. Level of complexity: need some experience trading on a crypto exchange. 

● My referral ID: 864206 
https://www.hotbit.io/register?ref=864206

**Comment by pjvargo43** [comment]
![20211202_214544](https://user-images.githubusercontent.com/59556028/144552121-87a6faac-6ba0-46a7-be20-98d59744ab67.jpg)

![Screenshot_20211202-214450_SOLO DEX](https://user-images.githubusercontent.com/59556028/144552141-9ad0657d-a846-4dbe-b221-3353f3efa145.jpg)

**Comment by joshmcl8** [comment]
I am also having this same issue. Did anyone manage to resolve this problem?

**Comment by pjvargo43** [comment]
The issue has not been resolved yet. Under Repositories, there are 2 contributors and with the undefined Sologenic staff that handles these issues, might be limited as well. If the number is only a hand full, they have there plate full. Also, not to mention, it is the Holiday season, which team members might be taking time off. Honestly, it might be awhile before it gets fixed.

**Comment by joshmcl8** [comment]
Ok I understand. It’s a good job the price is moving north else I think there would be a lot more complaints. Either way though this isn’t good having tokens stuck on an exchange

**Comment by pjvargo43** [comment]
I totally agree with you, 100%. I don't like to hold assets on exchanges because it's not yours until you have it in your personal wallet...but under the circumstances, I can't afford not to miss SOLO under < $10.00. SOLO's current price is a total steal before it goes double digits by the end of the year. Wooyee! Lol
